src/frontend/styles/Game.module.scss

```diff
@@ -87,7 +87,7 @@ top: 580.41px; */
 }
 
 .gameDisplay {
-  margin: 5% 0 10%;
+  margin: 5% 0 5%;
```

Comment:
I think `5% 0` is automatically `5% 0 5% 0`, but that's a nitpick

Reply:
Didn't know that! Thx~